app/utils/pubsub.js

```diff
@@ -49,8 +50,16 @@ var init = ()=>{
 };
 
 var getNewClient = ()=>{
-  var conf = JSON.parse(process.env.REDIS_CONN_JSON || '{}');
-  return new Redis(conf);
+  var options = JSON.parse(process.env.REDIS_CONN_JSON || '{}');
+  if (process.env.REDIS_CERTIFICATE_PATH) {
+    // for self signed cert of redis server
+    options.tls = { ca: [fs.readFileSync(process.env.REDIS_CERTIFICATE_PATH)] } ;
```

Comment:
Handle read error, or not found error. Check existence then parse will be cleaner

Reply:
fixed.